README.md

```diff
@@ -119,14 +117,14 @@ To create your own instances of `ExternallyAuthenticatedAuthFilterFactory` from
 
 2. Add the full class name of your subclass of `ExternallyAuthenticatedAuthFilterFactory` inside.
 
-# Development
+## Development
 
 - Publish new versions via:
 
         export BINTRAY_USER=<user>
```

Comment:
@al-jeyapal this needs to be changed to however publishing to maven central works as bintray (along with jcenter) has become depricated and read-only.

Reply:
@adam-kimpton yep, look at the following:
https://github.com/brightsparklabs/asanti/blob/ea69e8ff81eb46c1f26643219e77b74398ee3d5f/.github/workflows/publish.yml#L36

Basically set those 4 env vars, then call `./gradlew publishToMavenCentral`